src/cryptonote_basic/tx_extra.h

```diff
@@ -457,6 +457,7 @@ enum Decommission_Reason : uint16_t {
     timestamp_response_unreachable = 1 << 4,
     timesync_status_out_of_sync = 1 << 5,
     lokinet_unreachable = 1 << 6,
+    l2_height_out_of_sync = 1 << 7,
```

Comment:
There are a couple functions in `tx_extra.cpp` that also need to be updated to understand this value.